fdb-extensions/src/main/java/com/apple/foundationdb/async/RankedSet.java

```diff
@@ -267,17 +477,30 @@ public RankedSet(Subspace subspace, Executor executor) {
      */
     public CompletableFuture<Boolean> contains(ReadTransactionContext tc, byte[] key) {
         checkKey(key);
-        return containsCheckedKey(tc, key);
+        return countCheckedKey(tc, key).thenApply(c -> c != null && c > 0);
+    }
+
+    /**
+     * Count the number of occurrences of a key in the set.
+     * @param tc the transaction to use to access the database
+     * @param key the key to check for
+     * @return a future that completes to {@code 0} if the key is not present in the ranked set or
+     * {@code 1} if the key is present in the ranked set and duplicates are not counted or
+     * the number of occurrences if duplicated are counted separately
+     */
+    public CompletableFuture<Long> count(ReadTransactionContext tc, byte[] key) {
```

Comment:
If there were a separate `RankedMultiset` class, this would only be on it, though it seems well-defined everywhere.